spec.bs

```diff
@@ -1707,6 +1707,25 @@ Note: When trusted scoring signals fetches are not batched, |renderURLs|'s [=lis
 
 </div>
 
+<div algorithm>
+To <dfn>send report</dfn> given a [=URL=] |url|:
+
+  1. Let |request| be a new [=request=] with the following properties:
+    :   [=request/URL=]
+    ::  |url|
+    :   [=request/client=]
+    ::  `null`
+    :   [=request/mode=]
+    ::  "`no-cors`"

```

Comment:
Another `no-cors` here. Our implementation didn't change the [default mode](https://source.chromium.org/chromium/chromium/src/+/refs/heads/main:services/network/public/cpp/resource_request.h;drc=a5150e1903bde84e008ac27d8881d9ccf72e087e;l=161) of a resource request, which is `no-cors`.

Reply:
Then I suppose we should add a big TODO or red box here pointing to https://github.com/WICG/turtledove/issues/667, since this is generally a bad practice and should be under serious scrutiny wherever used.

Reply:
Added `Issue(WICG/turtledove#667)`, and set metadata `Inline Github Issues: title`, to avoid displaying the issue's quite long description.
One problem is I added this 5 times (for 5 `no-cors`), which seems fetching the github issue 5 times, then I we probably hit the github's rate limit error more often. Not sure if there's a way to avoid fetching the same issue multiple times, or maybe replace the issue(xxx) with a normal issue and a manual link to the github issue.

Reply:
Yeah that's what I meant by "normal issue". Probably that's better to do, to avoid it fetching github issue using github API and hit the rate limit (we only had [one such issue](https://github.com/WICG/turtledove/blob/main/spec.bs#L2795), but we've encountered the rate limit error a few times, and needed to wait 1 hour or some hours to rerun the github check).

Reply:
Done.